packaging/rpm/zrepl.spec

```diff
@@ -0,0 +1,100 @@
+# This Spec file packages pre-built artifacts in the artifacts directory
+# into an RPM. This means that rpmbuild won't actually build any new artifacts
+#
+# Read the Makefile first, then come back here.
+
+# Add workaround for "No build ID note found" error during %%install
+%undefine _missing_build_ids_terminate_build
+
+
+# Disable debuginfo for RedHat/CentOS builds to avoid errors due to empty
+# debuginfo files, see also: https://fedoraproject.org/wiki/Packaging:Debuginfo
+%if 0%{?rhel}
+%global debug_package %{nil}
+%endif
```

Comment:
Disable the debug packages everywhere, which further allows to remove the workaround for the build ID issue:
```suggestion
# Disable debug packages as the binaries are not built by this spec file, see also:
# https://fedoraproject.org/wiki/Packaging:Debuginfo
￼%global debug_package %{nil}
```